src/helpers/figures/charts/chart_common.ts

```diff
@@ -272,6 +276,24 @@ export function toExcelDataset(getters: CoreGetters, ds: DataSet): ExcelChartDat
     };
   }
 
+  let trend: ExcelChartTrendConfiguration | undefined = undefined;
+  if (ds?.trend?.type) {
+    trend = {
+      type: CHART_TRENDLINE_TYPE_CONVERSION_MAP_REVERSE[ds.trend.type],
+      order: ds?.trend?.order,
```

Comment:
You added a comment in another file saying the max degree in Excel is 6. Maybe we should check this here too ?